packages/larva-patterns/modules/vlanding-video-card/vlanding-video-card.twig

```diff
@@ -0,0 +1,45 @@
+{% if o_video_card_permalink_url %}
```

Comment:
Update all instances of `o-video-card` to `vlanding-video-card` and all instances of `o_video_card` to `vlanding_video_card`